bigframes/operations/semantics.py

```diff
@@ -61,30 +114,13 @@ def filter(self, instruction: str, model):
                 raise ValueError(f"Column {column} not found.")
 
         # Replace column references with names.
-        instruction = instruction.format(**{col: col for col in columns})
+        user_instruction = user_instruction.format(**{col: col for col in columns})
 
         prompt_df = self._df.copy()
```

Comment:
nit:  maybe we can just call `self._df[columns].copy()`? Though BF has optimizations already to skip unused columns in query.

Reply:
SG!